server/graphql/v2/query/CryptoExchangeRateQuery.js

```diff
@@ -0,0 +1,22 @@
+import { GraphQLNonNull, GraphQLString } from 'graphql';
+
+import { getCryptoFxRate } from '../../../lib/currency';
+
+const CryptoExchangeRateQuery = {
+  type: GraphQLString,
+  args: {
+    cryptoCurrency: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'Crypto currency symbol. Example: BTC',
+    },
+    collectiveCurrency: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'Fiat currency symbol. Example: USD',
+    },
+  },
+  async resolve(_, args) {
+    return getCryptoFxRate(args.cryptoCurrency, args.collectiveCurrency);
```

Comment:
Why do we need something specific to crypto? Could we consider something generic instead `exchangeRateQuery `